<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabResultLogWidget</class>
 <widget class="QWidget" name="LabResultLogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextEdit" name="edtLog"/>
   </item>
   <item>
    <widget class="QFrame" name="frmFind">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edtFind"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnNext">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnPrevious">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtLog</tabstop>
  <tabstop>edtFind</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>btnPrevious</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
